rfcs/password_validation_limits.md

```diff
@@ -19,6 +19,13 @@ A possible solution is to use existing password-strength checking solution like
 Enabling password validator is Breaking change and disabled by default.
 In the nearest future, when everything will be ready, a validator must be enabled.
 
+## OTP and Empty Password
+The `password` field is not defined as `required` in current `schemas`.
+Validation does not happen because This field does not exist in the OTP registration request.
+When the Service performs password generation,  `skipPassword` property does not exist in the registration request, so validation not executing on generated passwords.
```

Comment:
* double space before `skipPassword`
* <...> so the validator does not check these generated passwords.